src/chains/ethereum_1/ChainEthereumV1.ts

```diff
@@ -135,17 +151,53 @@ class ChainEthereumV1 implements Chain {
 
   // Chain Helper Functions
 
-  isValidEntityName = notImplemented
+  // --------- Chain helper functions
+
+  /** Verifies that the value is a valid chain entity name (e.g. an account name) */
+  public isValidEntityName = (value: string): boolean => {
+    return !!isValidEthereumEntityName(value)
```

Comment:
The !! operator transforms any value into boolean. If the source value is already a boolean, then you don't need to do !!. So this function should just return isValidEthereumEntityName(value) not !!isValidEthereumEntityName(value)